analysis/paladinholy/src/modules/items/UntemperedDedication.tsx

```diff
@@ -0,0 +1,111 @@
+import SPELLS from 'common/SPELLS';
```

Comment:
This isn't about this line but rather the whole file.
Why is it in the items folder when its a conduit? Wouldn't it make more sense to put it in a conduits folder... which i'd imagine would be under the shadowlands folder